_includes/hero.html

```diff
@@ -1,59 +1,57 @@
+    {% set hero_href = ['href=', hero.custom_fields.link[0]] %}
+{% endif %}
+
 <section class="content_hero hero"
          style="background-color: {{ hero.custom_fields.background_color[0]
                                      if hero.custom_fields.background_color }}">
     <div class="hero_wrapper">
-    {% if related_post %}
-        <a class="hero_card" href="{{ related_post.permalink }}">
-    {% elif hero.custom_fields.link %}
-        <a class="hero_card" href="{{ hero.custom_fields.link[0] }}">
-    {% else %}
-        <a class="hero_card">
-    {% endif %}
-            <div class="hero_card-bg">
-                <div class="hero_card-padding">
+        <a class="hero_card" {{ hero_href|join if hero_href }}>
+            <article class="hero_preview">
                 {% if related_post %}
-                    <span class="category-slug">
-                        {{ category_icon(related_post.category.0, 'category-slug_icon') }}
-                        <span class="u-visually-hidden">Category:</span>
-                        {{ related_post.category.0 }}
-                    </span>
+                <span class="category-slug">
+                    {{ category_icon(related_post.category.0, 'category-slug_icon') }}
+                    <span class="u-visually-hidden">Category:</span>
+                    {{- related_post.category.0 -}}
+                </span>
                 {% endif %}
-                    <div class="summary">
-                        <h1 class="summary_header">
-                            {{ hero.title }}
-                        </h1>
-                        <p class="summary_text">
-                        {% if hero.content %}
-                            {{ hero.content|striptags }}
-                        {% elif related_post %}
-                            {% if related_post.dek %}
-                                {{ related_post.dek|striptags }}
-                            {% else %}
-                                {{ related_post.excerpt|striptags }}
-                            {% endif %}
-                        {% endif %}
-                        </p>
-                    {% if related_post %}
-                        {% if related_post.author.0 %}
+                <div class="summary">
+                    <header class="summary_header">
+                        <h1 class="summary_heading">{{ hero.title }}</h1>
+                    </header>
+                    <p class="summary_text">
+                    {%- if hero.content %}
+                        {{- hero.content|striptags -}}
+                    {%- elif related_post %}
+                        {%- if related_post.dek %}
+                            {{- related_post.dek|striptags -}}
+                        {% else %}
+                            {{- related_post.excerpt|striptags -}}
+                        {% endif -%}
+                    {% endif -%}
+                    </p>
+                    <footer class="summary_footer">
```

Comment:
The events hero `footer` will contain different child content (some times `time`, some times a live feed `a`). Having one parent `footer` across any type of content makes it easier to target the styles no matter what's living inside. Added the `header` because I always find blocks with `footers` but no `headers` slightly off. I'm cool with removing it.